core/iwasm/libraries/wasi-nn/test/utils.c

```diff
@@ -108,7 +108,12 @@ run_inference(execution_target target, float *input, uint32_t *input_size,
               uint32_t num_output_tensors)
 {
     graph graph;
+
+#if WASM_ENABLE_WASI_EPHEMERAL_NN == 0
     if (wasm_load(model_name, &graph, target) != success) {
+#else
+    if (wasm_load_by_name(model_name, &graph) != success) {
+#endif
```

Comment:
In this PR, you can simply replace `wasm_load()` with ``wasm_load_by_name()` without involving `WASM_ENABLE_WASI_EPHEMERAL_NN`